assets/js/modules/adsense/components/settings/AdBlockingRecoveryApp.js

```diff
@@ -17,22 +17,93 @@
 /**
  * WordPress dependencies
  */
+import { __ } from '@wordpress/i18n';
 import { Fragment } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
+import Data from 'googlesitekit-data';
 import Header from '../../../../components/Header';
 import HelpMenu from '../../../../components/help/HelpMenu';
+import Layout from '../../../../components/layout/Layout';
+import { Cell, Grid, Row } from '../../../../material-components';
+import PageHeader from '../../../../components/PageHeader';
+import AdBlockingSetupSVG from '../../../../../svg/graphics/ad-blocking-recovery-setup.svg';
+import Link from '../../../../components/Link';
+import {
+	BREAKPOINT_SMALL,
+	useBreakpoint,
+} from '../../../../hooks/useBreakpoint';
+import { CORE_SITE } from '../../../../googlesitekit/datastore/site/constants';
+const { useSelect } = Data;
 
 export default function AdBlockingRecoveryApp() {
+	const breakpoint = useBreakpoint();
+
+	const settingsURL = useSelect(
+		( select ) =>
+			`${ select( CORE_SITE ).getAdminURL(
+				'googlesitekit-settings'
+			) }#/connected-services/adsense`
+	);
+
+	const isTabletWidthOrLarger = breakpoint !== BREAKPOINT_SMALL;
+
 	return (
 		<Fragment>
 			<Header>
 				<HelpMenu />
 			</Header>
 			<div className="googlesitekit-ad-blocking-recovery">
-				Ad Blocking Recovery
+				<div className="googlesitekit-module-page">
+					<Grid>
```

Comment:
I think we can either combine two divs into one or two divs with the grid to reduce nesting:

```js
<div className="googlesitekit-ad-blocking-recovery googlesitekit-module-page">
    <Grid>
```
or
```js
<Grid className="googlesitekit-ad-blocking-recovery googlesitekit-module-page">
```

Reply:
Fair enough; I'm copying an existing/similar layout but I'll give that a go 👍🏻

Reply:
We need the one div, but I've combined them 👍🏻